eya_def_tools/eya_def_tools/utils/doc_utils/eya_def_erdantic.py

```diff
@@ -1,129 +1,303 @@
-"""Adaptation of the erdantic package for the EYA DEF.
+"""Adaptation of the erdantic functionality for the EYA DEF.
```

Comment:
The `erdantic` package completely changed the API for plugins when going from v0 to v1, so I had to rewrite pretty much everything in this module. I took the opportunity to refine the diagram representations a bit. The type names are now clearer and JSON Schema type names are also included.